db_release_scripts/mosip_master/sql/1.2.0_master-scripts_release.sql

```diff
@@ -255,6 +260,16 @@ ALTER TABLE master.user_detail_h DROP COLUMN uin;
 ALTER TABLE master.user_detail_h DROP COLUMN email;
 ALTER TABLE master.user_detail_h DROP COLUMN mobile;
 
+SELECT * INTO master.template_copy FROM master.template;
+
+UPDATE master.template set module_id='10001' where module_id='10002' and template_typ_code not like 'reg-%';
```

Comment:
Please add a detailed comment on why we are doing this module id changes, since this is not directly related to migration but related to performance or cleanup. Lets include if there is any warning also.

Reply:
Done

Reply:
Resolved